submissions/peterIyiola/css/signup.css

```diff
@@ -0,0 +1,168 @@
+   .container   {
+        display: flex;
+        flex:row wrap;
+        justify-content: space-around;
+        padding:18px;
+    }
+    .flex-item  {
+        align-items: center;
+    }
+    .flex-item1 {
+        margin-left:20px;
+        align-items: center;
+    }
+    .logo   {
+        width: 30px;
+        height:30px;
+    }
+    .pp {
+        padding-top: 13px;
+    }
+    .CU {
+        padding-top: 13px;
+    }
+    .SU {
+        padding-top: 13px;
+    }
+    
+    .firstname  {
+        border-radius: 25px;
+    }
+    a   {
+        text-decoration: none;
+    }
+    .btn    {
+        border-radius: 1px;
+        color: goldenrod;
+        margin-left: 100px;
+        cursor: pointer;
+        width: 100px;
+        height: 30px;
+    }
+    
+    .btn:hover  {
+        background-color:burlywood;
+    }
+    .check  {
+        margin-left: 30px;
+    }
+    .Name   {
+        border-radius: 10px;
+        padding: 5px;
+        width: 300px;
+    }
+    .FS   {
```

Comment:
let your class names be descriptive what it is doing or where it is been used. @Iyiola-stack Rename the class in html as well

Reply:
Done